sycl/test-e2e/DeviceGlobal/device_global_arrow.cpp

```diff
@@ -1,7 +1,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 //
-// UNSUPPORTED: opencl && gpu
+// UNSUPPORTED: (opencl && gpu) || target-native_cpu
```

Comment:
This should be it's own line. I doubt it is covered by GSD-4287. :wink:

Same goes for the other DeviceGlobal tests.

Reply:
Also, should new issue in `intel/llvm` be created as an `// UNSUPPORTED-TRACKER:`? or is there existing `intel/llvm` issue that can be used?